workspace/spec/lita/handlers/teamcity_spec_anki.rb

```diff
@@ -93,4 +93,42 @@
     end
 
   end
+
+  describe '#list' do
+    build_type_json_str = '{"buildType":[{"id":"Wildcard_A","projectName":"Wildcard :: A"},
+                                         {"id":"B_Wildcard","projectName":"Wildcard :: B"}]}'
+    before do
+      stub_request(:get, /buildTypes/).to_return(:status => 200, :body => build_type_json_str)
+    end
+
+    it 'reply the builds if wildcard exists' do
+      expected_output = "Here are the builds I found:\n\n"
+                        "Wildcard :: A :: id=`Wildcard_A` \n\n"
+                        "Use a build `id` from the list to trigger a build. *@buildbot build `id`*"
+      send_command("list Wildcard_A*")
+      expect(replies.last).to include(expected_output)
+    end
+
+    it 'reply all builds if wildcard is empty' do
+      expected_output = "Here are the builds I found:\n\n"
+                        "Wildcard :: A :: id=`Wildcard_A`\n"
+                        "Wildcard :: B :: id=`B_Wildcard` \n\n"
+                        "Use a build `id` from the list to trigger a build. *@buildbot build `id`*"
+      send_command("list")
+      expect(replies.last).to include(expected_output)
+    end
+
+    it 'reply buildtypes.empty if wildcard invalid' do
```

Comment:
I think the description should be minor changed, e.g. 'replies buildtypes.empty if wildcard is invalid'